scripts/extract.py

```diff
@@ -224,7 +225,12 @@ def check_thread_error(self):
 
     def output_processing(self, faces, size, filename):
         """ Prepare faces for output """
-        self.align_face(faces, size, filename)
```

Comment:
Mask _base.py outputs an aligned face so this align code is redundant. Fold non-align related code into output_processing